cmake/Modules/Options.cmake

```diff
@@ -37,6 +38,23 @@ else()
     set(_USE_CUDA OFF)
 endif()
 
+################################################################################
+#
+#        OpenCV (required for CPU acceleration)
+#
+################################################################################
+
+set(OpenCV_COMPONENTS opencv_core opencv_imgproc)
+find_package(OpenCV QUIET COMPONENTS ${OpenCV_COMPONENTS})
```

Comment:
We shouldn't be looking for OpenCV or adding it to the linked libraries if it wasn't requested.